packages/pages/src/scaffold/modules/generate.ts

```diff
@@ -110,9 +113,12 @@ function handleCancel(moduleName: string, projectStructure: ProjectStructure) {
     fs.rmdirSync(modulePath);
   }
 
-  const modulesPath = path.join("src", "modules");
-  if (fs.readdirSync(modulesPath).length === 0) {
-    fs.rmdirSync(modulesPath);
+  const moduleDirPath = path.join(
```

Comment:
Won't this delete all modules if you cancel the scaffold? We'd only want to clean up THE module folders that were created by this command.

Reply:
In this case, it only removes the `modules/` dir if it is empty. But yeah I can remove it + have it only delete the `modules/` dir if was created by this command.